integration_tests/tests/apply/view_application_overview_as_referrer.cy.ts

```diff
@@ -0,0 +1,78 @@
+//  Feature: Referrer views a submitted application overview
+//    So that I can revisit a submitted application
+//    As an referrer
+//    I want to view the overview of a particular submitted application
+//
+//  Scenario: visits submitted application overview page
+//    Given a submitted application exists
+//    And I am logged in as a referrer
+//    When I visit the application overview for that application
+//    Then I see the submitted application overview
+//
+//  Scenario: first visit to submitted application overview page
```

Comment:
I think we can remove this text as we aren't testing this scenario.

Reply:
oh yeah, good spot :)